modules/ei_analytics_dashboard/templates/carbon-home/wso2/analytics/conf/dashboard/deployment.yaml.erb

```diff
@@ -0,0 +1,466 @@
+        maxPoolSize: 20
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+
+  # carbon metrics data source
+  - name: WSO2_METRICS_DB
+    description: The datasource used for dashboard feature
+    jndiConfig:
+      name: jdbc/WSO2MetricsDB
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: '<%= @metrics_db_url %>'
+        username: <%= @metrics_db_username %>
+        password: <%= @metrics_db_password %>
+        driverClassName: <%= @metrics_db_driver %>
+        maxPoolSize: 20
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+
+  - name: WSO2_PERMISSIONS_DB
+    description: The datasource used for dashboard feature
+    jndiConfig:
+      name: jdbc/PERMISSION_DB
+      useJndiReference: true
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: '<%= @permission_db_url %>'
+        username: <%= @permission_db_username %>
+        password: <%= @permission_db_password %>
+        driverClassName: <%= @permission_db_driver %>
+        maxPoolSize: 10
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+  - name: HTTP_ANALYTICS_DB
+    description: The datasource used for HTTP Analytics dashboard
+    jndiConfig:
+      name: jdbc/HTTP_ANALYTICS_DB
+      useJndiReference: false
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/worker/database/HTTP_ANALYTICS_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;MVCC=TRUE;AUTO_SERVER=TRUE'
+        username: wso2carbon
+        password: wso2carbon
+        driverClassName: org.h2.Driver
+        maxPoolSize: 30
+        idleTimeout: 60000
+        connectionTestQuery: SELECT 1
+        validationTimeout: 30000
+        isAutoCommit: false
+#  - name: Twitter_Analytics
+#    description: The datasource used for Twitter Analytics dashboard
+#    jndiConfig:
+#      name: jdbc/Twitter_Analytics
+#      useJndiReference: false
+#    definition:
+#      type: RDBMS
+#      configuration:
+#        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/worker/database/Twitter_Analytics;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;MVCC=TRUE;AUTO_SERVER=TRUE'
+#        username: wso2carbon
+#        password: wso2carbon
+#        driverClassName: org.h2.Driver
+#        maxPoolSize: 30
+#        idleTimeout: 60000
+#        connectionTestQuery: SELECT 1
+#        validationTimeout: 30000
+#        isAutoCommit: false
+  - name: Message_Tracing_DB
+    description: "The datasource used for message tracer to store span information."
+    jndiConfig:
+      name: jdbc/Message_Tracing_DB
+    definition:
+      type: RDBMS
+      configuration:
+        jdbcUrl: 'jdbc:h2:${sys:carbon.home}/wso2/dashboard/database/MESSAGE_TRACING_DB;AUTO_SERVER=TRUE'
```

Comment:
Shall we parameterize this as well? For the worker, this has already being done.